lib/Frontend/CompilerInvocation.cpp

```diff
@@ -42,6 +42,23 @@ void CompilerInvocation::setMainExecutablePath(StringRef Path) {
   setRuntimeResourcePath(LibPath.str());
 }
 
+/// If we haven't explicitly passed -prebuilt-module-cache-path, set it to
+/// the default value of <resource-dir>/<platform>/prebuilt-modules.
+static void updatePrebuiltModuleCachePath(
+  FrontendOptions &frontendOpts, const SearchPathOptions &searchPathOpts,
+  const llvm::Triple &triple) {
+
+  if (!frontendOpts.PrebuiltModuleCachePath.empty())
```

Comment:
This seems like it could be a problem if this is called more than once. Maybe that just means the function should be called something other than "update"?